apps/examples/src/components/SpanLink.tsx

```diff
@@ -6,7 +6,7 @@ import { useMergedRefs } from '../hooks/useMegedRefs'
 import { StandaloneIcon } from './Icons'
 import { Markdown } from './Markdown'
 
-export const ListLink = forwardRef(function ListLink(
+export const SpanLink = forwardRef(function ListLink(
```

Comment:
may i suggest actually just calling this Link?
in the future it might not be a `span` (but, let's say a div) and we wouldn't want to rename this again :)